CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v0.8.0] WIP
```

Comment:
We also need links at the bottom.

Reply:
Good call - I added links and fixed the previous versions.
I linked to PRs because the repo does not have folders or github releases.